safekeeper/src/state.rs

```diff
@@ -149,7 +150,7 @@ impl TimelinePersistentState {
             &TenantTimelineId::empty(),
             Configuration::empty(),
             ServerInfo {
-                pg_version: 170000, /* Postgres server version (major * 10000) */
+                pg_version: PgVersionId::from_full_pg_version(170000), /* Postgres server version (major * 10000) */
```

Comment:
I think you could remove this comment since I assume that docs `from_full_pg_version()` and `PgVersionId` cover it

Reply:
Made obsolete by using PgMajorVersion + PgVersionId::from() directly.